src/components/essential/DlTypography/DlTypography.vue

```diff
@@ -124,5 +136,20 @@ export default defineComponent({
     // important needed above for sizing of h tags.. quasar conflict
     // to deal with quasar conflicts.
     line-height: initial !important;
+
+    // These are the classes that are used to limit the number of lines a kpi-title spans
+    &--number-of-lines-1 {
```

Comment:
Are the options only 1 or 2 lines?
If so, the prop shouldn't just accept a number - it should be a boolean to allow multiline.

If it should support a dynamic number of number of rows, handle it accordingly